documentation/docs/steps/transportRequestUploadCTS.md

```diff
@@ -0,0 +1,173 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have installed the SAP component SAP_UI 7.53 or higher on your ABAP system.
+* You have enabled the OData Service to Load Data to the [SAPUI5 ABAP Repository](https://sapui5.hana.ondemand.com/#/topic/a883327a82ef4cc792f3c1e7b7a48de8.html)
+* You have the [S_DEVELOP authorization](https://sapui5.hana.ondemand.com/#/topic/a883327a82ef4cc792f3c1e7b7a48de8.html) for operations on a SAPUI5 ABAP repository.
+* You have created a transport request on the ABAP system, which is the target container of the SAPUI5 application to upload.
+
+## Upload Client Setup
+
+The step `transportRequestUploadCTS` uses the [Node.js](https://nodejs.org)-based [SAP Fiori tools](https://help.sap.com/viewer/product/SAP_FIORI_tools/Latest/en-US) to upload your SAPUI5 application to the UI5 repository service of your ABAP back-end infrastructure. It performs a deployment running the [Fiori deploy command](https://www.npmjs.com/package/@sap/ux-ui5-tooling#fiori-deploy---performs-the-deployment-of-the-application-into-an-abap-system) on a Docker image.
```

Comment:
it performs the deployment command [Fiori deploy](https://www.npmjs.com/package/@sap/ux-ui5-tooling#fiori-deploy---performs-the-deployment-of-the-application-into-an-abap-system)*